react/utils/assets.ts

```diff
@@ -12,9 +12,9 @@ export const getVTEXImgHost = (account: string) => {
   return `https://${account}.vteximg.com.br`
 }
 
-const getAbsoluteURL = (account: string, url: string, workspace: string, production: boolean) => {
+const getAbsoluteURL = (account: string, url: string, production: boolean) => {
   return isRelative(url) && production
-    ? `${getVTEXImgHost(account)}${url}?workspace=${workspace}`
+    ? `${getVTEXImgHost(account)}${url}`
```

Comment:
If the workspace is in production, we should fetch the corresponding asset, right ?